src/components/contexts/MapContextProvider.tsx

```diff
@@ -248,9 +236,7 @@ const MapContextProvider: FC = ({children}) => {
   const [areStopsVisible, setAreStopsVisible] = useState(false);
   const [propertyGrid, setPropertyGrid] = useState<Stop | Conversation | null>(null);
   const [rerenderConnections, setRerenderConnections] = useState(false);
-  const [connectionData, setConnectionData] = useState<
-    Array<{coordinates: {lat: number; lon: number}; id: string; name: string; ref: string; isOsm: boolean}>
-  >([]);
+  const [connectionData, setConnectionData] = useState<Array<Stop>>([]);
```

Comment:
`<Array<Stop>>` is equivalent to  `<Stop[]>`. I personally like to use `Array` only when there is more code and there is possibility for square brackets to disappear among code mess. It is just a suggestion.

Reply:
Stop[] is better. I'll fix it tomorrow.